tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_passcode.py

```diff
@@ -17,9 +17,12 @@ def test_set_passcode_by_settings(self):
         screen_lock_settings = settings.open_screen_lock_settings()
 
         screen_lock_settings.enable_passcode_lock()
+        old_passcode_code = self.data_layer.get_setting(
+            'lockscreen.passcode-lock.digest.value')
         screen_lock_settings.create_passcode(self._input_passcode)
 
         passcode_code = self.data_layer.get_setting('lockscreen.passcode-lock.code')
         passcode_enabled = self.data_layer.get_setting('lockscreen.passcode-lock.enabled')
-        self.assertEqual(passcode_code, "".join(self._input_passcode), 'Passcode is "%s", not "%s"' % (passcode_code, "".join(self._input_passcode)))
+        self.assertNotEqual(passcode_code, old_passcode_code, 'Passcode did not change.')
```

Comment:
Duh. yep.

The problem is that with a hashed passcode, we can't really verify it in the clear :-)

My idea was to *at least* notice that the passcode setting was somehow changed (the defaults already contain a passcode). But verifying using the UI seems like a better idea. AFAIU there are already other tests which do this, should I merge them or does this too warrant writing up a function and calling it in all the places where a UI based check happens?